canonicalwebteam/blog/blog_api.py

```diff
@@ -135,6 +135,11 @@ def _transform_article(self, article):
                 article["content"]["rendered"]
             )
 
+            # use cloudinary for blog article images
+            article["content"]["rendered"] = self._replace_img_to_cloudinary(
```

Comment:
Aren't you gonna make this optional through a parameter somewhere? Otherwise this kinda does the exact opposite of what is asked for in https://github.com/canonical-web-and-design/canonicalwebteam.blog/issues/139

Reply:
It is optional under the same parameter `self.transform_links` which decides the admin.insights url rewrites.

I should have created a new one.

Reply:
No it's ok for it to be both I think. But the `admin.insights.ubuntu.com` one doesn't actually matter that much, maybe that doesn't even need to be optional - because no-one else will be using that domain, and we will *always* want to replace it I think.

Reply:
I created a second one `enable_cloudinary`. I can remove the `transform_links`, but I don't think it harms being there.

Reply:
I don't feel strongly about the existence of this one, but I do generally consider unnecessary config options to be harmful. They are cognitive load for the developer to process, extra documentation for us to write, extra logic in the codebase...

Reply:
Oh also, we're potentially confusing our own Devs, cos they might mistakenly think there are some cases where they shouldn't be doing this replacement, when in fact they should always do it.

Reply:
I added a new commit to remove `transform_links`. `enable_cloudinary` is the only new parameter. It controls the cloudinary prefix for both article images and thumbnails.